pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -615,10 +622,11 @@ def create_server_instance(connection_name):
     resolved_pull_max_cnt = pull_max_cnt or DEFAULT_MAXPULLCNT
     resolved_timeout = timeout or DEFAULT_CONNECTION_TIMEOUT
 
-    # Command mode (ctx is None). Processes command on comand line and quits
+    # Command mode (ctx is None) or initial input in interactive mode.
     # Apply the documented option defaults to create a pywbem_server instance
     # and a ContextObj instance
-    if ctx.obj is None:  # No context. This is cmd line mode
+    if ctx.obj is None:  # No context. This is cmd line mode or initial input
+        # in interactive mode.
```

Comment:
These are unrelated changes.

Reply:
No change, just clarification of comment.  It is and always been not just the cmd line mode but also called at the beginning of the interactive mode.

Reply:
That was a surprise to me also.

Reply:
On "No change, just clarification of comment": It is still a change.
I am not saying the change is invalid, I am saying it is unrelated to what this PR intends to do.
If you want to do change things that are unrelated to what this PR intends to do, it should ideally go into a separate PR. I know this is hard to do for small changes... Let's leave that in for now, but let's consider that for the future.